examples/panel/book-recommender/rag_book_recommender.py

```diff
@@ -0,0 +1,89 @@
+import os
```

Comment:
i'd recommend using pathlib.Path instead of os.path.join, it's cleaner

Reply:
[fixed](https://github.com/ploomber/doc/pull/127/files#diff-396e0215830b7ad98dc178d3fd362e3e0a577ab3509a784b915a8242e092be98R40-R41)